examples/bigcommerce/.env

```diff
@@ -0,0 +1,7 @@
+BIGCOMMERCE_STORE_NAME=
+BIGCOMMERCE_STORE_HASH=
+BIGCOMMERCE_CLIENT_SECRET=
+BIGCOMMERCE_ACCESS_TOKEN=
+BIGCOMMERCE_CLIENT_ID=
+
+MAKESWIFT_SITE_API_KEY=
```

Comment:
Missing newline at the end of the file. Also, `MAKESWIFT_SITE_API_KEY` and other env vars that should be secrets should probably be kept our of the `.env` file and instead be configured on the Vercel deployment.

Reply:
Fair enough. My intention was for this to become their `.env.local` in the case that they cloned the repo. Should i rename it to `.env.example`

Reply:
Yes. Putting this like this in a `.env` will cause all of these env vars to be empty strings which might cause unintended side effects. Also documenting the env vars in the README and having good error messages when running the app works as well. Having a centralized config module can help with that.